cms/static/js/views/active_video_upload_list.js

```diff
@@ -173,9 +173,7 @@ define(
             },
 
             clearSuccessful: function() {
-                var completedIndexes = [];
-                var completedModels = [];
-                var idx;
+                var completedIndexes = [], completedModels = [], idx;
```

Comment:
@tanmaykm to get the eslint `one-var-declaration-per-line` error to go away you just need to move this `completedModels = []` to another line like you have above.

```javascript
var idx,
    completedIndexes = [],
    completedModels = [];
```